pkg/commands/tls/subscription/subscription_test.go

```diff
@@ -51,6 +51,62 @@ func TestTLSSubscriptionCreate(t *testing.T) {
 			Args:       "--domain example.com",
 			WantOutput: fmt.Sprintf("Created TLS Subscription '%s' (Authority: %s, Common Name: example.com)", mockResponseID, certificateAuthority),
 		},
+		{
+			Name: "validate cert-auth == certainly",
+			API: mock.API{
+				CreateTLSSubscriptionFn: func(i *fastly.CreateTLSSubscriptionInput) (*fastly.TLSSubscription, error) {
+					return &fastly.TLSSubscription{
+						ID:                   mockResponseID,
+						CertificateAuthority: i.CertificateAuthority,
+						CommonName:           i.Domains[0],
+					}, nil
+				},
+			},
+			Args:       "--domain example.com --cert-auth certainly",
+			WantOutput: fmt.Sprintf("Created TLS Subscription '%s' (Authority: certainly, Common Name: example.com)", mockResponseID),
+		},
+		{
+			Name: "validate cert-auth == lets-encrypt",
+			API: mock.API{
+				CreateTLSSubscriptionFn: func(i *fastly.CreateTLSSubscriptionInput) (*fastly.TLSSubscription, error) {
```

Comment:
This should really be refactored into a function, but I'll leave that for our future team members who will be working on improving the test suite.